frontend/src/pages/ClientDetailsPage.vue

```diff
@@ -44,14 +81,47 @@ import { toTitleCase } from "@/services/ForestClientService";
           This div is necessary to avoid the div.header-offset below from interfering in the flex flow.
         -->
         <div data-scroll="top-notification" class="header-offset"></div>
+        <cds-actionable-notification
```

Comment:
We have a page for this notifications. Can you call it instead of putting this code in this page?
The page is called ErrorNotificationGroupingComponent.vue

Reply:
Yes, thanks. I did it like that because I used a different kind of condition to trigger it. But I'll try to accommodate or unify the code.

Reply:
I have created [FSADT1-1669](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT1-1669) for dealing with this.